lib/alchemy/engine.rb

```diff
@@ -14,6 +14,10 @@ class Engine < Rails::Engine
       NonStupidDigestAssets.whitelist += [/^tinymce\//]
     end
 
+    initializer 'alchemy.yaml_whitelist_classes' do
+      Alchemy::YAML_WHITELIST_CLASSES = %w(Symbol Date Regexp)
+    end
```

Comment:
This makes it hard to read the Alchemy config before this initializer runs. Why not put this into the `alchemy.rb` module definition?

Also I think moving this constant into a yaml reading service class would be the most preferable solution.

Reply:
Agreed, I will move it. Also agree on the service class solution, but not within this PR :)

Reply:
👍